### PR TITLE
Revised ground-mounted PV measure

### PR DESCRIPTION
**Pull request by srgilani**
revised pv measure and necb2011 to consider building footprint as default area for pv panels, and set necb_default for pv_ground_type as do-nothing for pv measure, and added none option for all args of pv measure

### CODE REVIEW

File: test/necb/unit_tests/expected_results/pv_ground_expected_results.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "building_type": "FullServiceRestaurant",
     "primary_heating_fuel": "DefaultFuel",
     "pv_ground_type": true,
-    "Generator PVWatts 1 - dc_system_capacity": "8611.12833336778",
```
Comment (phylroy) [inline test/necb/unit_tests/expected_results/pv_ground_expected_results.json:8]: wow big change.. was this a correction from mike?

File: lib/openstudio-standards/standards/necb/ECMS/pv_ground.rb [hunk 2]
```diff
@@ -2,12 +2,52 @@ class ECMS
...
+      pv_ground_azimuth_angle = pv_ground_azimuth_angle.to_f
+    end
+
+    ##### Calculate footprint of the building model (this is used as default value for pv_ground_total_area_pv_panels_m2)
+    building_footprint_m2 = 0.0
+    model.getSpaces.sort.each do |space|
+      space.surfaces.sort.each do |surface|
+        if surface.surfaceType == 'Floor' && surface.outsideBoundaryCondition == 'Ground'
+          floor_vertices = surface.vertices
+          if floor_vertices[0].z == 0.0
```
Comment (phylroy) [inline lib/openstudio-standards/standards/necb/ECMS/pv_ground.rb:40]: What would happen if the ground floor was not at exactly zero?

**Review by phylroy — CHANGES_REQUESTED** [review]

File: lib/openstudio-standards/standards/necb/ECMS/pv_ground.rb (hunk 2 above)
Comment (srgilani) [reply lib/openstudio-standards/standards/necb/ECMS/pv_ground.rb]: Revised to the lowest floors' area (assuming the floor of all lowest spaces are at the same level).

File: test/necb/unit_tests/expected_results/pv_ground_expected_results.json (hunk 1 above)
Comment (srgilani) [reply test/necb/unit_tests/expected_results/pv_ground_expected_results.json:8]: No, previously I set the default area of the PV panels as 50 m2, whereas I set it to a building's footprint in my revisions.

**Review by phylroy — APPROVED** [review]

**Pull request merged**